code/game/machinery/bots/secbot.dm

```diff
@@ -243,6 +243,9 @@ Auto Patrol: []"},
 		if (istype(C, /mob/living/carbon/human))
 			threatlevel = assess_perp(C)
 
+		if (C.alpha == 1 || C.invisibility > SEE_INVISIBLE_LEVEL_TWO)
```

Comment:
shouldn't this be C.alpha == 0

Reply:
`		if(wielded)
			user.visible_message("<span class='danger'>\The [user] throws \the [src] over \himself and disappears!</span>","<span class='notice'>You throw \the [src] over yourself and disappear.</span>")
			user.register_event(/event/moved, src, .proc/mob_moved)
			user.alpha = 1	//to cloak immediately instead of on the next Life() tick
			user.alphas[CLOAKINGCLOAK] = 1`

Reply:
bizarre